aea/aea_builder.py

```diff
@@ -939,9 +938,7 @@ def run_build_for_component_configuration(
             logger.info(f"Building package {config.component_id}...")
 
         source_directory = cast(str, config.directory)
-        target_directory = os.path.abspath(
-            os.path.join(root_dir, config.build_directory)
-        )
+        target_directory = os.path.abspath(config.build_directory)
```

Comment:
I think this might be incorrect

Reply:
that's correct, config_build directory set with respect to root_build_directory set in AEABuilder